internal/events/types/hype_train/hype_train_end_event.go

```diff
@@ -0,0 +1,103 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package hype_train_end
+
+import(
+	"encoding/json"
+	"time"
+
+	"github.com/twitchdev/twitch-cli/internal/events"
+	"github.com/twitchdev/twitch-cli/internal/models"
+	"github.com/twitchdev/twitch-cli/internal/util"
+)
+
+var transportsSupported = map[string]bool{
+	models.TransportWebSub:   false,
+	models.TransportEventSub: true,
+}
+
+var triggerSupported = []string{"hype-train-end"}
+
+var triggerMapping = map[string]map[string]string{
+	models.TransportEventSub: {
+		"hype-train-end":        	"channel.hype_train.end",
+	},
+}
+
+type Event struct{}
+
+func (e Event) GenerateEvent(params events.MockEventParameters) (events.MockEventResponse, error) {
+	var event []byte
+	var err error
+
+	//Some values
+	StartedAtTime := util.GetTimestamp()
+	localTotal := util.RandomViewerCount()
+	localLevel := util.RandomViewerCount()%4
+	localRandomUser2 := util.RandomUserID()
+	localRandomUser3 := util.RandomUserID()
+	localTC := []models.ContributionData{{TotalContribution: util.RandomViewerCount(), TypeOfContribution: util.RandomType(), UserWhoMadeContribution: localRandomUser2, UserNameWhoMadeContribution: localRandomUser2, UserLoginWhoMadeContribution: localRandomUser2},{TotalContribution: util.RandomViewerCount(), TypeOfContribution: util.RandomType(), UserWhoMadeContribution: localRandomUser3, UserNameWhoMadeContribution: localRandomUser3, UserLoginWhoMadeContribution: localRandomUser3}}
+	EndedAtTime := util.GetTimestamp()
+	CooldownAtTime := util.GetTimestamp()
```

Comment:
Additionally, these should use RFC3339 for WebSub and RFC3339Nano for EventSub. For examples, see the other topics.